src/evaluation/Run.jl

```diff
@@ -47,7 +47,8 @@ function run_reactive!(session::ServerSession, notebook::Notebook, old_topology:
 	end
 
 	# Send intermediate updates to the clients at most 20 times / second during a reactive run. (The effective speed of a slider is still unbounded, because the last update is not throttled.)
-	send_notebook_changes_throttled = throttled(1.0/20, 0.0/20) do
+	# flush_send_notebook_changes_throttled, 
+	send_notebook_changes_throttled, flush_notebook_changes = throttle(1.0/20; leading=false, trailing=true) do
 		send_notebook_changes!(ClientRequest(session=session, notebook=notebook))
```

Comment:
This code will run at most once per 1/20 sec, `@async`. 
BUT there is no guarantee I can think of that [X] code will not run right before/right after/inbetween